README.md

```diff
@@ -1009,7 +1009,7 @@
     }
     ```
 
-  - Use camelCase when naming objects, functions, and instances
+  - Use camelCase when naming objects, functions, and instances. Abbreviations and acronyms should also use camel case when used as a name.
```

Comment:
Acronyms and initialisms should be properly capitalized - ie, `XML`, not `Xml` - since the purpose is readability, not appeasing an automatic camelcase parsing algorithm. Can you revert this line?